posixlib/src/main/scala/scala/scalanative/runtime/SocketHelpers.scala

```diff
@@ -46,7 +45,8 @@ object SocketHelpers {
 
         fcntl(sock, F_SETFL, O_NONBLOCK)
 
-        val fdsetPtr = selectFdSet.createZeroed
+        // Zone.alloc is documented as returning zeroed memory.
+        val fdsetPtr = alloc[fd_set].cast[Ptr[fd_set]]
```

Comment:
`alloc[fd_set]` does return `Ptr[fd_set]`, no need to cast it after.

Reply:
`alloc[fd_set]` neat feature!  So that is at least part of what the macro is doing.

cast removed. comments tweaked. Thank you for the recommendation.  Travis CI 
shows all green.